tests/helpers/container-query.ts

```diff
@@ -1,11 +1,28 @@
 import { find } from '@ember/test-helpers';
 
-export default function setupContainerQueryTest(hooks) {
+type DataAttributes = {
+  [dataAttributeName: string]: string | undefined;
+};
+
+type Features = {
+  [featureName: string]: boolean | undefined;
+};
+
+export interface CustomAssert extends Assert {
+  areDataAttributesCorrect?: (dataAttributes: DataAttributes) => void;
+  areDimensionsCorrect?: (
+    expectedWidth: number,
+    expectedHeight: number
+  ) => void;
+  areFeaturesCorrect?: (features: Features) => void;
+}
```

Comment:
I couldn't find online an example of how to type custom assertions. By trial and error, I came up with this interface. The `?:` was needed for the `delete` operator not to cause a TypeScript error.